### PR TITLE
Remove salt referances

### PR DESCRIPTION
**Pull request by oms4suse**
Remove lots of salt references in the code base.

This is due to the code originally being a salt library but its now not dependent on salt.

### CODE REVIEW

File: ceph_cfg/__init__.py [hunk 1]
```diff
@@ -188,37 +168,19 @@ def osd_prepare(**kwargs):
...
-        set the file system to store OSD data with. Defaults to "xfs".
-
-    osd_uuid
-        set the OSD data UUID. If set will return if OSD with data UUID already exists.
-
-    journal_uuid
-        set the OSD journal UUID. If set will return if OSD with journal UUID already exists.
+    Args:
+        **kwargs: Arbitrary keyword arguments.
+            cluster_uuid : Set the deivce to store the osd data on.
```
Comment (theanalyst) [inline ceph_cfg/__init__.py]: device

Comment (theanalyst) [inline ceph_cfg/__init__.py]: replace deivce/device for all cluster_uuid lines

File: ceph_cfg/__init__.py [hunk 2]
```diff
@@ -291,27 +242,13 @@ def keyring_purge(**kwargs):
...
-    cluster_uuid
-        Set the cluster UUID. Defaults to value found in ceph config file.
-
-    cluster_name
-        Set the cluster name. Defaults to "ceph".
-
-    If no ceph config file is found, this command will fail.
+    Args:
+        **kwargs: Arbitrary keyword arguments.
+            keyring_type : Required paramter. Can be set to: admin, mon, osd,
```
Comment (theanalyst) [inline ceph_cfg/__init__.py]: parameter

Comment (theanalyst) [inline ceph_cfg/__init__.py]: similar for future occurences

**Comment by oms4suse** [comment]
Thanks for the review I have made corrections where you suggested.

**Comment by theanalyst** [comment]
lgtm

**Pull request merged**